orderapi/src/main/java/fi/hel/verkkokauppa/order/service/invoice/InvoiceService.java

```diff
@@ -15,14 +18,27 @@ public class InvoiceService {
 
     final InvoiceMapper invoiceMapper;
 
+    final IncrementId incrementId;
```

Comment:
Could you rename this field to incrementIdGenerator etc, this caused some thinking on my side when first saw this

Reply:
Yes, incrementId name is misleading for what the field and class is used for - I'll change this.